includes/controls/animation.php

```diff
@@ -59,7 +59,27 @@ protected function get_default_settings() {
 	 * @return array Control type.
 	 */
 	public static function get_animations() {
-		$animations = [
```

Comment:
I have extracted the $animations to a method called `get_default_animations()`, so that I can access that method inside the `assets/loader.php` file as well.